.editorconfig

```diff
@@ -8,6 +8,7 @@ tab_width=4
 indent_style=space
 indent_size=4
 tab_width=4
+csharp_prefer_braces=true:hint
```

Comment:
Ahh, thank you for this!

Reply:
It has its flaws:
https://github.com/blish-hud/Blish-HUD/pull/290/commits/1656d70e9b39b3119117460934d8602aa2c746d8#diff-867794a57e040a77f0b88399e707dbbbR104-R106

Reply:
😕 Well, I suppose that's collateral damage that can maybe be addressed with further tweaking of the config.  In any case, I still appreciate you throwing that in.  👍